romfs/lang/de.json

```diff
@@ -25,15 +25,15 @@
         },
         "applet": {
             "title": "Applet-Modus wird nicht unterstützt",
-            "desc": "Es kann zu Problemen bei der Verwendung des Awoo-Installers im Applet-Modus kommen.\nWenn du Probleme hast, wechsele bitte dazu über, den Awoo Installer über einen installierten Titel zu starten (halte R gedrückt, während du ein Spiel startest)!"
+            "desc": "Es kann zu Problemen bei der Verwendung des Awoo Installer's im Applet-Modus kommen.\nWenn du Probleme hast, starte den Awoo Installer über einen installierten Titel (halte R gedrückt, während du ein Spiel startest)!"
```

Comment:
"Installer's" is wrong.
German doesn't have the possessive apostrophe like English does.
Also while not really wrong, I'd change the sentence to `Es kann zu Problemen bei der Verwendung von Awoo-Installer im Applet-Modus kommen.` as that sounds more natural.

Reply:
I can change it, and remove the apostrophe with the 's'
I found it optical better with the apostrophe 😉